specification/offazurespringboot/resource-manager/Microsoft.OffAzureSpringBoot/preview/2023-01-01-preview/springbootdiscovery.json

```diff
@@ -1789,6 +1893,7 @@
         },
         "provisioningState": {
           "type": "string",
+          "readOnly": true,
           "description": "The resource provisioning state.",
           "enum": [
             "Unknown",
```

Comment:
>readOnly [](http://example.com/codeflow?start=11&length=8)

Breaking change.

Reply:
The provisionState should be readonly by the validation rules